### ISSUE TITLE
Better NetworkPacket chunking

### ISSUE THREAD

**Issue by leafreynolds** [leafreynolds/cosmere#105]
There's a lot of data on the player's spiritweb and often we just send the entire thing (regardless of what's changed) to every client. We probably do this via multiple triggers: general syncing, mode changes, etc.

We should refine this, with the ability to mark certain sections of the spiritweb as needing re-syncing. That way we can drastically reduce the number of packets and size of those packets.

- [ ] Power Modes
- [ ] Submod sections (ie allomancy consumed metal counts, hemalurgy eye position etc)
- [ ] Changing selected manifestation
etc etc.

Basically mark things as only being sent if they have changed.
